packages/cfd/src/Containers/mt5-desktop-download-option.tsx

```diff
@@ -0,0 +1,94 @@
+                {desktopDownloadOptions.map((option, index) => (
+                    <div
+                        key={`${option.icon}-${index}`}
+                        className={`cfd-trade-modal__download-center-app--option ${
+                            index > 0 ? 'cfd-trade-modal__download-center-app--option-hide' : ''
+                        }`}
+                    >
+                        <Icon icon={option.icon} size={32} />
+                        <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
+                            {option.text}
+                        </Text>
+                        <a
+                            className='dc-btn cfd-trade-modal__download-center-app--option-link'
+                            type='button'
+                            href={option.href}
+                            target='_blank'
+                            rel='noopener noreferrer'
+                        >
+                            <Text size='xxs' weight='bold' color='prominent'>
+                                {option.button_text}
+                            </Text>
+                        </a>
+                    </div>
+                ))}
+            </div>
+            <Text align='center' as='div' className='cfd-trade-modal__download-center-text' size='xxs' weight='bold'>
+                {localize(
+                    'Download {{ platform }} on your phone to trade with the {{ platform }} {{ account }} account',
+                    {
+                        platform: getCFDPlatformLabel(CFD_PLATFORMS.MT5),
+                        account: account_title,
+                    }
+                )}
+            </Text>
+            <div className='cfd-trade-modal__download-center-options'>
+                <div className='cfd-trade-modal__download-center-options--mobile-links'>
+                    <a
+                        href={mt5_trade_account?.white_label?.download_links?.ios}
+                        target='_blank'
+                        rel='noopener noreferrer'
+                    >
+                        <Icon icon='IcInstallationApple' width={135} height={40} />
+                    </a>
+                    <a
+                        href={mt5_trade_account?.white_label?.download_links?.android}
+                        target='_blank'
+                        rel='noopener noreferrer'
+                    >
+                        <Icon icon='IcInstallationGoogle' width={135} height={40} />
+                    </a>
+                    <a href={getPlatformMt5DownloadLink('huawei')} target='_blank' rel='noopener noreferrer'>
+                        <Icon icon='IcInstallationHuawei' width={135} height={40} />
+                    </a>
+                </div>
+
+                <div className='cfd-trade-modal__download-center-options--qrcode cfd-trade-modal__download-center-options--qrcode-hide'>
```

Comment:
Shouldn't you use a condition for the hide class ?

Reply:
just noticed, this classname is removed from the css there is `--qrcode` but no `--qrcode-hide`
<img width="390" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/a0251c68-4bca-43f2-883f-e2e5f737b190">